admin/src/main/java/itcast/application/AdminNewsService.java

```diff
@@ -31,19 +33,40 @@ public AdminNewsResponse createNews(Long userId, News news) {
 
         return new AdminNewsResponse(savedNews);
     }
-
+  
     public Page<AdminNewsResponse> retrieveNews(Long userId, NewsStatus status, LocalDate sendAt, int page, int size) {
         isAdmin(userId);
         Pageable pageable = PageRequest.of(page, size);
         return newsRepository.findNewsBYCondition(status, sendAt, pageable);
     }
 
+    @Transactional
+    public AdminNewsResponse updateNews(Long userId, Long newsId, AdminNewsRequest adminNewsRequest) {
+        isAdmin(userId);
+        News news = newsRepository.findById(newsId).orElseThrow(()->
+                new IdNotFoundException("해당 뉴스가 존재하지 않습니다"));
```

Comment:
줄바꿈은 개인 차이가 있을 거 같은데, 전체적으로 `orElseThrow`에서 줄바꿈하는 편인 거 같아서 한번 수정해보시면 좋을 거 같습니다.
```suggestion
        News news = newsRepository.findById(newsId)
                .orElseThrow(()-> new IdNotFoundException("해당 뉴스가 존재하지 않습니다"));
```